recipes/daal4py/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = '2020.2' %}
+{% set buildnumber = 0 %}
+
+package:
+    name: daal4py
+    version: {{ version }}
+
+source:
+    url:    https://github.com/IntelPython/daal4py/archive/{{ version }}.tar.gz
+    sha256: d71028538d028f1cc03706d9b1984ac932da45592696edd678518954ac485c4f
+build:
+    skip: True  # [osx]
+    number: {{ buildnumber }}
+    include_recipe: False
+    script_env:
+     - DPCPPROOT
+     - DAALROOT
+     - TBBROOT
```

Comment:
Why? These should be removed

Reply:
Thanks for review. This is copy of recipe used for building daal4py and vars used there of override of those 3 tools. 
For conda recipe we can remove them - will do in repo.

The only problem is DPCPPROOT - so basically soon we will need DPCPP compiler in order to build daal4py with GPU support.  Can we get DPCPP native integration in build environment so it can be selected? There is no conda package for compiler - only runtimes.   
Or we can follow current approach then compiler is installed in ci and we use it during build?  DPCPP compiler is free and avaialbe via apt/other channels

https://intel.github.io/llvm-docs/GetStartedGuide.html

Reply:
@napetrov We have an interest in shipping the full intel compiler stack for C/C++/FORTRAN, including DPCPP support, if that is possible. We'd then allow people to add builds to their feedstocks with the intel compilers. What do you think?

cc @mariusvniekerk

Reply:
@beckermr - think this would be useful for overall comunity.  I've start internal discussion on feasibility and next steps

Reply:
@beckermr  - looks there is no restrictions on using compiles now and Intel compilers can be used from oneAPI packages.
Is there any expectation on how this should be done? doing trough repack - in this case we should have packages at Intel channel 

or thisis handled in different way for compilers?

Reply:
Wow! This is great!

We should make our own compiler packages here. We'll need our own flags for the builds and to make sure they integrate properly with the rest of our stack. 

Can you join the next dev meeting to talk about this?

bump @conda-forge/core

Reply:
@beckermr  - yes sure, this depends on time of meeting although as i'm in UTC+3 timezone. Can you share invite? 
And guess @tomashek will be also interested

Reply:
Our dev meeting is at 1700 UTC. Do you have an email address to send the invite to?

Reply:
nikolay.a.petrov@intel.com

Reply:
OK I sent a note to you. Hopefully @tomashek can join too!

Reply:
@beckermr Hi there! Can you send an invite to andres.guzman-ballen@intel.com so I can be a part of these discussions? Also, can we can touch base via that same email to continue the conversation over the best approach to providing the community Intel compiler conda packages? Thanks!

Reply:
Hi! 

We are currently discussing in a public gitter room: https://gitter.im/conda-forge/conda-forge-compilers

Feel free to join us there. We have had a few email exchanges, but those are mostly dead now. I can forward them along to you separately.